dotCMS/src/main/java/com/dotcms/security/apps/SecretsKeyStoreHelper.java

```diff
@@ -341,27 +311,17 @@ private String getCustomKeyProvider() {
      * @return
      */
     @VisibleForTesting
-    protected char[] encrypt(final char[] val) {
+    char[] encrypt(final char[] val) {
```

Comment:
Codacy found an issue: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)